README.md

```diff
@@ -75,6 +75,11 @@ Now you will see a review with linting errors...
     $ trivy --quiet fs -f json path/to/directory/ |lintly --format=trivy
     ```
 
+-  [tfsec](https://github.com/tfsec/tfsec)
+    ```
+    $ tfsec path/to/directory/ -f json |lintly --format=tfsec
```

Comment:
Please add a space between the `|` and `lintly`! (e.g., `| lintly --format=tfsec`)

(And fix the other examples that are missing this space.)

Reply:
This has been repeated feedback -- let's try and get this space in there before merging this PR

Reply:
Yes on it!